### PR TITLE
Competition Leg

### PR DESCRIPTION
**Pull request by BipinKalra**
Competition Cards and concentric circles.

### CODE REVIEW

File: sass/styles/components/competition-card.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.competition-card{
```
Comment (vdvibhu20) [inline sass/styles/components/competition-card.scss]: c-prefix

File: sass/styles/components/faq-card.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.faq-card{
```
Comment (vdvibhu20) [inline sass/styles/components/faq-card.scss]: c-prefix

File: sass/styles/utilities/color.scss [hunk 3]
```diff
@@ -69,13 +69,17 @@ $gradient-list: (black, 'lr', linear-gradient(30deg, #333333, #000000)),
...
-  background-image: linear-gradient(to right, #193b84, #2675bb);
+  background-image: $bg-gradient-blue-lr;
 }
 
 .bg-gradient-purple{
-  background: linear-gradient(30deg, #271147, #553180);
+  background: $bg-gradient-purple;
 }
 
 .bg-gradient-orange{
```
Comment (vdvibhu20) [inline sass/styles/utilities/color.scss]: nice

File: sass/styles/components/img-ring.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.img-ring{
```
Comment (vdvibhu20) [inline sass/styles/components/img-ring.scss]: add paddin css here

File: examples/components/hb/competition-carousel.hbs [hunk 5]
```diff
@@ -0,0 +1,139 @@
+<div class="border-card p-5">
+  <h5 class="t-align-c">Archived CodSule Contests</h5>
+  <div class="font-sm t-align-c mt-2 mb-5">These contests are already ended. You can still participate, however you will
+    not be awarded any points for these contest.</div>
+  <div class="row rating-carousel">
+    <div class="col-lg-3 col-md-4 col-6">
```
Comment (vdvibhu20) [inline examples/components/hb/competition-carousel.hbs]: make it col-8 and default padding

File: examples/components/hb/competition-carousel.hbs [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+          <a href="#" class="orange">
+            Participate
+            <i class="fas fa-arrow-right mx-3"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+    <div class="col-lg-3 col-md-4 col-6">
+      <div class="border-card p-xl-4 p-3 bg-gradient-dblue white">
+        <div class="font-mds bold">CodSule - April 2019</div>
```
Comment (vdvibhu20) [inline examples/components/hb/competition-carousel.hbs]: use respective <h> tag to reduce font-mds class.

File: examples/components/hb/up-competition-card.hbs [hunk 7]
```diff
@@ -0,0 +1,44 @@
+<div class="img-card competition-card">
```
Comment (vdvibhu20) [inline examples/components/hb/up-competition-card.hbs]: c- prefix

File: examples/components/hb/up-competition-card.hbs [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  <div class="head d-flex bg-gradient-dblue py-4 px-3 px-md-2 justify-content-center align-items-center">
```
Comment (vdvibhu20) [inline examples/components/hb/up-competition-card.hbs]: py-4 px-3 px-md-2 justify-content-center align-items-center 
these classes won't be needed once you use `.content` inside `.head`

File: examples/views/competition-banner.hbs [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+        </div>
+        <div class="description t-align-l col-11">
+          <div class="font-sm bold">Maharaja Agresen Institute Of Technology</div>
+          <span>40000 Points</span>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-7 col-12">
+      <h5 class="mb-4">Top Students</h5>
+      <div class="img-desc align-items-center justify-content-center orange my-4">
```
Comment (vdvibhu20) [inline examples/views/competition-banner.hbs]: no need of `.justify-content-center`

File: examples/views/competition-banner.hbs [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+          <h2>#1</h2>
+        </div>
+        <div class="col-xl-2 col-md-3 col-2">
+          <div class="img-ring s-50x50">
+            <img class="round w-100 h-100"
+              src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTmL43mSCo0MAu3fCgV8t6QRsNzyiatE_Har4TjE8M6NZ-i0tKDzA"
+              alt="">
+          </div>
+        </div>
+        <div class="description t-align-l col-xl-9 col-md-8 col-9">
```
Comment (vdvibhu20) [inline examples/views/competition-banner.hbs]: no need of `t-align-l col-xl-9 col-md-8 col-9`
just some padding left or padding right on img-ring

File: examples/views/competition-card.hbs [hunk 11]
```diff
@@ -0,0 +1,30 @@
+<div class="row">
+  <div class="col-md-4 col-12">
+    <div class="img-card c-competition-card">
+      <div class="head d-flex bg-gradient-dblue py-4 px-3 px-md-2 justify-content-center align-items-center">
```
Comment (vdvibhu20) [inline examples/views/competition-card.hbs]: unnecessary classes `d-flexm justify-content-center align-items-center`

File: examples/views/competition-card.hbs [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+        <div class="d-flex flex-column t-align-c">
```
Comment (vdvibhu20) [inline examples/views/competition-card.hbs]: no need of d-flex flex-column

File: examples/components/hb/competition-banner.hbs [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+        <div class="col-1 pl-0">
+          <h2>#1</h2>
+        </div>
+        <div class="description t-align-l col-11">
+          <div class="font-sm bold">Bharati Vidyapeeth College of Engineering</div>
+          <span>43000 Points</span>
+        </div>
+      </div>
+      <div class="img-desc justify-content-center my-4">
+        <div class="col-1 pl-0">
```
Comment (vdvibhu20) [inline examples/components/hb/competition-banner.hbs]: no need of `col-` classes

File: examples/components/hb/competition-banner.hbs [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+        </div>
+        <div class="description t-align-l col-11">
+          <div class="font-sm bold">Maharaja Agresen Institute Of Technology</div>
+          <span>40000 Points</span>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-7 col-12">
+      <h5 class="mb-4">Top Students</h5>
+      <div class="img-desc align-items-center justify-content-start orange my-4">
```
Comment (vdvibhu20) [inline examples/components/hb/competition-banner.hbs]: justify-content-start not required

File: examples/components/hb/competition-banner.hbs [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+          <h2>#1</h2>
+        </div>
+        <div class="pl-xl-2 pl-md-3 pl-0">
+          <div class="img-ring s-50x50">
+            <img class="round w-100 h-100"
+              src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTmL43mSCo0MAu3fCgV8t6QRsNzyiatE_Har4TjE8M6NZ-i0tKDzA"
+              alt="">
+          </div>
+        </div>
+        <div class="description t-align-l col-10">
```
Comment (vdvibhu20) [inline examples/components/hb/competition-banner.hbs]: t-align-l col-10 not required

Comment (BipinKalra) [reply examples/components/hb/competition-banner.hbs]: already fixed.

File: examples/views/up-competition-card.hbs [hunk 16]
```diff
@@ -0,0 +1,48 @@
+<div class="row">
+  <div class="col-md-4 col-12">
+    <div class="img-card c-competition-card">
+      <div class="head d-flex bg-gradient-dblue py-4 px-3 px-md-2 justify-content-center align-items-center">
```
Comment (vdvibhu20) [inline examples/views/up-competition-card.hbs]: remove align-items-center

File: examples/views/up-competition-card.hbs [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+        <div class="d-flex flex-column t-align-l">
+          <div class="font-mds">Upcoming Contest</div>
+          <h2>CodSule - June 2019</h2>
+          <ul class="divided-list t-align-c">
```
Comment (vdvibhu20) [inline examples/views/up-competition-card.hbs]: no need of t-align classes in divided list

**Pull request merged**